Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -160,26 +173,13 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
 
     private lazy var adjustedRatingLabel: UILabel = .build { label in
         let text = String.Shopping.ReviewQualityCardAdjustedRatingLabel
```

Comment:
This label gets its text assigned twice. Please remove this occurrence so that only the attributed text is set to the label.